### PR TITLE
contrib/eza: new package (0.17.0)

### PR DESCRIPTION
**Pull request by nekopsykose**

### CODE REVIEW

File: contrib/eza/template.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+hostmakedepends = [
+    "cargo",
+    "pkgconf",
+]
+makedepends = [
+    "libgit2-devel",
+    "rust-std",
+    "zlib-devel",
+]
+pkgdesc = "Modern ls replacement"
```
Comment (q66) [inline contrib/eza/template.py:14]: pkgdesc bad

File: contrib/eza/template.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+pkgname = "eza"
+pkgver = "0.17.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [
```
Comment (q66) [inline contrib/eza/template.py:5]: probably one line

File: contrib/eza/template.py (hunk 1 above)
Comment (nekopsykose) [reply contrib/eza/template.py:14]: what would you like it to be

Comment (q66) [reply contrib/eza/template.py:14]: make something up, but Subjective Adjectives shouldn't go in pkgdescs; pkgdesc should be neutral and descriptive ("Directory listing utility" or whatever)

Comment (nekopsykose) [reply contrib/eza/template.py]: when they're that small it kinda sounds really weird for some reason, but okie

Comment (q66) [reply contrib/eza/template.py]: i mean it's just as small as the previous one

**Pull request merged**